src/promptbase/gsm8k/gsm8k.py

```diff
@@ -62,7 +67,7 @@ def solve(idx):
             break
 
     if answer:
-        with open("gsm8k.jsonl", "a") as f:
+        with open(my_path.parent / "datasets" / "gsm8k.jsonl", "a") as f:
```

Comment:
I'm not sure we want to do this.... it's starting to look to me as if the `gsm8k.jsonl` is just the intermediate results?

Reply:
Some of our runs do have intermediate results that need to be stored somewhere. We might want to put them in their own designated shared directory somewhere.